### DISCUSSION TITLE
Docker image for OL 9.2

### DISCUSSION THREAD

**Discussion by jennisms** [oracle/container-images#60] [Q&A]
Hello,

I'm looking for a base image of OL 9.2. I know that only the latest dot releases are officially supported. But having some kind of access to older images would be helpful. Is there a way to still get the 9.2 image?

Thanks, Jennis

**Comment by tvierling** [comment]
No, there is not.

This is a fundamental, albeit common, misconception of how the version of Oracle Linux works.

There is no such thing as a "dot release." The version of the OS and packages is "Oracle Linux 9," full stop. No dot or number following.

The only thing the number after the dot tells you is "these packages are *at least as new* as the corresponding ISO install media." It means nothing else.

Because this misconception caused many issues in the past with users misusing OL6/7/8 images, we dropped the concept of images tagged with a dot as of Oracle Linux 9. There is only one image stream, and only ever has been.
